File: 백준/Silver/17207. 돌려막기/돌려막기.py
if __name__ == '__main__':
    people = ["Inseo", "Junsuk", "Jungwoo", "Jinwoo", "Youngki"]
    A = [[0] * 5 for _ in range(5)]
    B = [[0] * 5 for _ in range(5)]
    for i in range(5):
        A[i] = list(map(int, input().split()))

    for i in range(5):
        B[i] = list(map(int, input().split()))
    C = [[0] * 5 for _ in range(5)]
    for x in range(len(C)):
        for y in range(len(C[i])):
            for i in range(5):
                C[x][y] += A[x][i] * B[i][y]
    sums = []
    for c in C:
        sums.append(sum(c))
    min = min(sums)
    for i in range(len(sums)-1, -1, -1):
        if sums[i] == min:
            print(people[i])
            break

File: 백준/Silver/17207. 돌려막기/돌려막기.swift
var A = Array(repeating: [Int](), count: 5)
var B = Array(repeating: [Int](), count: 5)
let people = ["Inseo", "Junsuk", "Jungwoo", "Jinwoo", "Youngki"]
for i in 0..<5 {
    A[i] = readLine()!.split(separator: " ").map { Int($0)! }
}
for i in 0..<5 {
    B[i] = readLine()!.split(separator: " ").map { Int($0)! }
}

var C = Array(repeating: Array(repeating: 0, count: 5), count: 5)

for x in 0..<5 {
    for y in 0..<5 {
        for i in 0..<5 {
            C[x][y] += A[x][i] * B[i][y]
        }
    }
}

var sums = [Int]()

for row in C {
    sums.append(row.reduce(0, +))
}

let min = sums.min()

for i in (0..<5).reversed() {
    if min == sums[i] {
        print(people[i])
        break
    }
}